client/lib/plans/plans-list.js

```diff
@@ -393,13 +393,16 @@ const getPlanJetpackSecurityDailyDetails = () => ( {
 	group: constants.GROUP_JETPACK,
 	type: constants.TYPE_SECURITY_DAILY,
 	getTitle: () =>
-		getJetpackCROActiveVersion() === 'v2'
-			? translate( 'Jetpack Security {{em}}Daily{{/em}}', { components: { em: <em /> } } )
-			: translate( 'Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+		( {
+			v2: translate( 'Jetpack Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+			spp: translate( 'Jetpack Security' ),
+		}[ getJetpackCROActiveVersion() ] ||
+		translate( 'Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ) ),
 	getButtonLabel: () =>
-		getJetpackCROActiveVersion() === 'v2'
-			? translate( 'Get Security {{em}}Daily{{/em}}', { components: { em: <em /> } } )
-			: undefined,
+		( {
+			v2: translate( 'Get Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+			spp: translate( 'Get Jetpack Security' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **14** times:
`translate( 'Get Jetpack Security <em>Daily</em>' )` *ES Score: 6*